pallets/inflation/src/tests.rs

```diff
@@ -0,0 +1,233 @@
+#![cfg(test)]
+#![allow(clippy::from_over_into)]
+use crate as pallet_inflation;
+
+use frame_support::{
+	assert_ok, parameter_types,
+	traits::{Currency, OnInitialize, Everything, ConstU32},
+};
+use frame_system::RawOrigin;
+use sp_core::H256;
+use sp_runtime::{
+	traits::{BlakeTwo256, BlockNumberProvider, IdentityLookup},
+	testing::Header,
+};
+
+use sp_runtime::{
+	Perbill,
+};
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+const YEAR: u64 = 2_629_800; // 12-second blocks
+const FIRST_YEAR_BLOCK_INFLATION: u64 = 950;
+
+parameter_types! {
+	pub const ExistentialDeposit: u64 = 1;
+	pub const MaxLocks: u32 = 50;
+}
+
+impl pallet_balances::Config for Test {
+	type AccountStore = System;
+	type Balance = u64;
+	type DustRemoval = ();
+	type Event = ();
+	type ExistentialDeposit = ExistentialDeposit;
+	type WeightInfo = ();
+	type MaxLocks = MaxLocks;
+	type MaxReserves = ();
+	type ReserveIdentifier = ();
+}
+
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		Balances: pallet_balances::{Pallet, Call, Storage},
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		Inflation: pallet_inflation::{Pallet, Call, Storage},
+	}
+);
+
+parameter_types! {
+	pub const BlockHashCount: u64 = 250;
+	pub BlockWeights: frame_system::limits::BlockWeights =
+		frame_system::limits::BlockWeights::simple_max(1024);
+	pub const SS58Prefix: u8 = 42;
+}
+
+impl frame_system::Config for Test {
+	type BaseCallFilter = Everything;
+	type BlockWeights = ();
+	type BlockLength = ();
+	type DbWeight = ();
+	type Origin = Origin;
+	type Call = Call;
+	type Index = u64;
+	type BlockNumber = u64;
+	type Hash = H256;
+	type Hashing = BlakeTwo256;
+	type AccountId = u64;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type Header = Header;
+	type Event = ();
+	type BlockHashCount = BlockHashCount;
+	type Version = ();
+	type PalletInfo = PalletInfo;
+	type AccountData = pallet_balances::AccountData<u64>;
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type SystemWeightInfo = ();
+	type SS58Prefix = SS58Prefix;
+	type OnSetCode = ();
+	type MaxConsumers = ConstU32<16>;
+}
+
+parameter_types! {
+	pub FutureAuctionTreasuryId: u64 = 1111;
```

Comment:
Can we update the test to use AccountId type instead of u64?

Reply:
Or even better to adapt the pallet to accept palletId and then the pallet converts into account